lib/srv/desktop/rdp/rdpclient/src/vchan.rs

```diff
@@ -26,25 +26,33 @@ use std::io::{Cursor, Read};
 /// (or a single chunk) over a virtual channel.
 /// See https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpbcgr/343e4888-4c48-4054-b0e3-4e0762d1993c
 /// for more information about chunks.
+///
+/// To prevent memory DoS attacks that could take down the entire system running the Windows Desktop Service,
+/// Client drops all RDP messages > capacity.
```

Comment:
1. Capacity here is an internal implementation detail, so I wouldn't mention it at the Client level. It should be mentioned int he docs for the `new` fn.
2. You don't have to say "to prevent memory DoS attacks that could take down the whole system." Just say "client will drop all messages with length greater than the specified capacity.